truffle-config.js

```diff
@@ -13,6 +13,27 @@
  */
 
 module.exports = {
+  networks: {
+    development: {
+      host: "localhost",
+      port: 8545,
+      network_id: "*"
+    }
+  },
   // See <http://truffleframework.com/docs/advanced/configuration>
   // to customize your Truffle configuration!
+  // Configure your compilers
+  compilers: {
+    solc: {
+      version: "v0.6.2",    // Fetch exact version from solc-bin (default: truffle's version)
```

Comment:
Any reason not to use the latest version?

Reply:
That's just simply the version I'm currently using haha. And I think should check against close to v0.6 rather than latest for backward/low version support.